react-native-libraries.json

```
@@ -15912,5 +15912,13 @@
     "android": true,
     "newArchitecture": true,
     "web": true
+  },
+  {
+    "githubUrl": "https://github.com/vega-ui/ui-react-native/tree/main/packages/ui",
```

Hey @SlDo, thanks for adding new entry to the directory! 👍 

Just one suggestion, let's also list the example app, which is present within the repo:
```suggestion
    "githubUrl": "https://github.com/vega-ui/ui-react-native/tree/main/packages/ui",
    "examples": ["https://github.com/vega-ui/ui-react-native/tree/main/apps/preview"],
```